.github/workflows/stackhpc-container-image-build.yml

```diff
@@ -171,6 +171,9 @@ jobs:
           if ${{ inputs.push }} == 'true'; then
             args="$args --push"
           fi
+          if [[ "${{ github.event.inputs.regexes }}" == "" ]]; then
+            args="$args -e kolla_build_neutron_ovs=true"
```

Comment:
Could you instead set this in `etc/kayobe/environments/ci-builder/stackhpc-ci.yml`?

Reply:
Will it be something like

Reply:
Will it be something like
```
kolla_build_neutron_ovs: {{ true | if github.event.inputs.regexes == "" | default(omit) }}
```
?

Reply:
No, Kayobe/Ansible can't see the GitHub variables. It can be `true` unconditionally.

Reply:
If a regex is specified it will override the big one defined in kayobe.

https://opendev.org/openstack/kayobe/src/branch/master/ansible/container-image-build.yml#L14